src/screens/SubmitRatingScreen.tsx

```diff
@@ -21,10 +23,25 @@ const SubmitRatingScreen: React.FC<SubmitRatingScreenProps> = ({
 	placeName,
 	photo_reference,
 	setPage,
+	previousRating,
 }: SubmitRatingScreenProps) => {
 	const [counter, setCounter] = useState(
 		PLACE_ATTRIBUTES.reduce<{ [attribute: string]: number }>(function (countersMap, attribute) {
-			countersMap[attribute] = DEFAULT_INTERIM_RATING;
+			if (previousRating) {
+				// this smells bad
+				if (attribute === "Navigability") {
+					countersMap[attribute] = previousRating.navigability ?? DEFAULT_INTERIM_RATING;
+				} else if (attribute === "Sensory Aids") {
+					// TODO: change braille to sensory aids
+					countersMap[attribute] = previousRating.braille ?? DEFAULT_INTERIM_RATING;
+				} else if (attribute === "Staff Helpfulness") {
+					countersMap[attribute] =
+						previousRating.staffHelpfulness ?? DEFAULT_INTERIM_RATING;
+				} else if (attribute === "Guide Dog Friendliness") {
+					countersMap[attribute] =
+						previousRating.guideDogFriendly ?? DEFAULT_INTERIM_RATING;
```

Comment:
I'm unhappy with how this section turned out, but I'm not sure how I should approach this problem. do you all have any suggestions?

Reply:
I'm not sure what else you could do to make this smell less other than moving the logic to its own function and then calling it inside of here. But that would just move the ugly to a different place. tbh I probably would have written it the same way, curious if @aej11a or. @jlora23 have any ideas?

Reply:
I'm okay with refactoring this later, I think for now, it is totally okay @carpenterd777

Reply:
thanks. sorry to leave this in here